docs/04-tutorial.md

```diff
@@ -141,15 +141,16 @@ npm i -D rollup
 Now you can run it inside the root folder of your project like so:
 
 ```console
-./node_modules/.bin/rollup --config
+npm run rollup --config
+// OR yarn rollup --config
```

Comment:
This, however, seems to be correct as yarn does not seem to discern as stubbornly between scripts and executables.

Reply:
Yeah, this just works with yarn